pkg/kubelet/cm/container_manager.go

```diff
@@ -115,6 +119,12 @@ type ContainerManager interface {
 
 	// UpdateAllocatedDevices frees any Devices that are bound to terminated pods.
 	UpdateAllocatedDevices()
+
+	// GetAllDevices returns information about all the devices known to the manager
+	GetAllDevices() map[string]map[string]pluginapi.Device
```

Comment:
we should update this for a more descriptive type response in the future than a map of maps to devices.

Reply:
yes map[string]map[string]pluginapi.Device isn't so readable as e.g. DevicesPerResourcesMap or just DeviceResourceMap or ResourceDeviceMap.
that time it was release pressure, but now I think we can think about more readable name.